charts/seed-controlplane/charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -47,6 +47,9 @@ spec:
         garden.sapcloud.io/role: controlplane
         app: etcd-statefulset
         role: {{ .Values.role }}
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-public-except-private-medatadata: allowed
+        networking.gardener.cloud/from-prometheus: allowed
```

Comment:
Ingress from prometheus is also allowed  in `allow-from-apiserver-to-etcd`. And `from-prometheus` require port to be exposed under name `metrics`. I'm not sure if k8s allow to list container ports with same protocal and port under different names. If yes, then can we have metrics port exposed here as follows and  keep `allow-from-apiserver-to-etcd` specific to apiserver.  If not, then remove this label and rename the above network policy please.:

```yaml
- name: metrics
   containerPort: {{ .Values.servicePorts.client }}
   protocol: TCP
```